samples/lspci.c

```diff
@@ -55,11 +56,23 @@ int main(void)
                      PCI_HEADER_TYPE_NORMAL, 0) < 0) {
         err(EXIT_FAILURE, "vfu_pci_init() failed");
     }
-    vsc->vsc.hdr.id = PCI_CAP_ID_VNDR;
-    vsc->vsc.size = 0x10;
-    if (vfu_pci_setup_caps(vfu_ctx, caps, 2) < 0) {
-        err(EXIT_FAILURE, "failed to setup PCI capabilities");
+
+    pm.hdr.id = PCI_CAP_ID_PM;
```

Comment:
nit: why not initialize during variable declaration?

Reply:
done